src/main/kotlin/org/stellar/walletsdk/util/CreateDecoratedSignature.kt

```diff
@@ -5,6 +5,17 @@ import org.stellar.sdk.KeyPair
 import org.stellar.sdk.xdr.DecoratedSignature
 import org.stellar.sdk.xdr.Signature
 
+/**
+ * Create decorated signature using account's Stellar address (public key) and signature string.
+ *
+ * @param publicKey Stellar address of the account to use in the signature
```

Comment:
Related to my comment above, I think we should call this param `accountAddress` (or something similar) since we refer to public keys as "addresses" in other instances in the SDK